src/waveform/renderers/waveformmarkset.h

```diff
@@ -19,13 +21,16 @@ class WaveformMarkSet {
     void clear();
 
     int size() const { return m_marks.size();}
-    WaveformMark& operator[] (int i) { return m_marks[i]; }
+    QSharedPointer<WaveformMark> operator[] (int i);
```

Comment:
This methods are used to modify the WaveformMark. Should they be labeled const if they modify one of the objects WaveformMarkSet stores?